Source/Scene/Cesium3DTile.js

```diff
@@ -526,6 +526,19 @@ define([
                     return this._contentReadyPromise.promise;
                 }
             }
+        },
+
+        /**
+         * Returns the number of draw commands used by this tile.
+         *
+         * @readonly
+         *
+         * @private
+         */
+        commandsLength : {
```

Comment:
There are a couple places still referencing `._commandsLength` from outside of this class.